client/modules/css/src/components/_code.scss

```diff
@@ -26,7 +26,8 @@ pre.code {
 
 // -- TOKEN COLORS
 
-.wj-code-token {
+// weird selector is so we get priority over the default FTML selectors
+.wj-code-token.wj-code-token {
```

Comment:
Is this something we should change in the HTML generation?

Reply:
Nah, this is just something you have to deal with when doing CSS. It's unfortunate, but you get used to it.

Reply:
What does this mean for theme creators seeking to override this rule?

Reply:
Well, if they wanted to override these selectors, they'll need yet more priority. However, the whole point of this particular bit of CSS is to assign CSS variables to all of these different tokens, so a theme creator just needs to change the variables.